packages/SwingSet/package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "build": "exit 0",
     "test": "ava",
+    "test:xs": "exit 0",
```

Comment:
it's kinda boring that lerna isn't smart enough to skip packages with no such script.